Tests/SpeziTests/StandardTests/StandardInjectionTests.swift

```diff
@@ -51,16 +43,17 @@ final class StandardInjectionTests: XCTestCase {
     func testModuleFlow() async throws {
         let expectation = XCTestExpectation(description: "Module")
         expectation.assertForOverFulfill = true
-        
         let standardInjectionTestApplicationDelegate = await StandardInjectionTestApplicationDelegate(
             expectation: expectation
         )
         _ = await standardInjectionTestApplicationDelegate.spezi
-        
         await fulfillment(of: [expectation], timeout: 0.01)
     }
     
     func testInjectionPrecondition() throws {
+        guard !ProcessInfo.isReleaseTest else {
+            throw XCTSkip()
+        }
```

Comment:
Interesting; I wasn't aware that this doesn't work in release mode.

Reply:
Makes sense; we can address this once we move to 6.2 in the future, probably good to add some inline comments for this and/or a String passed into XCSkip that explains this.